### PR TITLE
enhance: add lazyload global config

### PR DESCRIPTION
**Pull request by sunby**

### CODE REVIEW

File: pkg/util/paramtable/component_param.go [hunk 1]
```diff
@@ -2178,6 +2180,14 @@ func (p *queryNodeConfig) init(base *BaseTable) {
 	}
 	p.MmapEnabled.Init(base.mgr)
 
+	p.LazyLoadEnabled = ParamItem{
+		Key:          "queryNode.lazyloadEnabled",
+		Version:      "2.4.0",
```
Comment (jaime0815) [inline pkg/util/paramtable/component_param.go:2185]: 2.4.1?

**Review by congqixia — APPROVED** [review]
/lgtm

**Pull request merged**